src/Lykke.Service.Assets/Modules/CacheModule.cs

```diff
@@ -58,21 +46,24 @@ protected override void Load(ContainerBuilder builder)
                 .As<ICachedErc20TokenAssetService>()
                 .SingleInstance();
 
-            RegisterRabbitMqSubscribers(builder);
+            RegisterRedis(builder);
 
-            builder.RegisterInstance(_settings.CurrentValue.AssetsService.RadisSettings)
-                .As<IAssetsForClientCacheManagerSettings>()
-                .SingleInstance();
+            RegisterCache<IAsset, AssetDto>(builder, "Assets");
+            RegisterCache<IAssetCategory, AssetCategory>(builder, "AssetCategories");
+            RegisterCache<IAssetPair, AssetPair>(builder, "AssetPairs");
+            RegisterCache<IErc20Token, Erc20Token>(builder, "Erc20Tokens");
 
             builder.RegisterType<AssetsForClientCacheManager>()
                 .As<IAssetsForClientCacheManager>()
-                .SingleInstance();
+                .SingleInstance()
+                .WithParameter(TypedParameter.From(_settings.RedisSettings.Expiration))
+                .WithParameter(TypedParameter.From($"{_settings.RedisSettings.Instance}:v3:ClientSettings"));
         }
 
         private void RegisterRedis(ContainerBuilder builder)
         {
-            System.Threading.ThreadPool.SetMinThreads(100, 100);
-            var options = ConfigurationOptions.Parse(_settings.CurrentValue.AssetsService.RadisSettings.RedisConfiguration);
+            System.Threading.ThreadPool.SetMinThreads(300, 300);
+            var options = ConfigurationOptions.Parse(_settings.RedisSettings.Configuration);
             options.ReconnectRetryPolicy = new ExponentialRetry(3000, 15000);
 
             var redis = ConnectionMultiplexer.Connect(options);
```

Comment:
set PreserveAsyncOrder = false;
https://stackoverflow.com/a/30912888/8592182

Reply:
This option is deprecated.